packages/studio-web/src/app/editor/segments/index.js

```diff
@@ -0,0 +1,176 @@
+import { Segment } from "./segment.js";
```

Comment:
this `app/editor/segments/` directory needs a 3rd party Copyright notice, I'll add it in a PR on top of this PR.